systemtest/src/main/java/io/strimzi/systemtest/resources/ResourceManager.java

```diff
@@ -235,6 +236,17 @@ private void createKafkaNodePoolIfNeeded(ExtensionContext testContext, Kafka res
 
             setNamespaceInResource(testContext, resource);
 
+            boolean nodePoolAlreadyExists = KafkaNodePoolResource.kafkaNodePoolClient().inNamespace(resource.getMetadata().getNamespace()).list().getItems()
+                .stream()
+                .anyMatch(knp -> {
+                    Map<String, String> labels =  knp.getMetadata().getLabels();
+                    return labels.containsKey(Labels.STRIMZI_CLUSTER_LABEL) && labels.get(Labels.STRIMZI_CLUSTER_LABEL).equals(resource.getMetadata().getName());
+                });
```

Comment:
This seems weird. What if you will have tests with multiple node pools? (which is one of the main points of node pools?) I think you need to use deterministic names instead of this hacky matching through labels. Not sure you need to change it right now in this PR. But I think you will have to sooner or later.

Reply:
thanks , we were considering both options, but in order to keep most tests unimpacted by move to kafkanodePools we wanted to skip this "automatic" creation of KafkaNodePool (together with Kafka Resource) in case somebody already binded some KafkaNodePool with this instance.